superset-frontend/package-lock.json

```diff
@@ -7,6 +7,7 @@
     "": {
       "name": "superset",
       "version": "0.0.0-dev",
+      "hasInstallScript": true,
```

Comment:
I think this should only be set if the `package.json` file has an "install" script in the `scripts` section, which we don't.